src/server/lib/okta/validateEmail.ts

```diff
@@ -1,54 +1,83 @@
+/* eslint-disable functional/immutable-data */
 import { updateUser } from '@/server/lib/okta/api/users';
-import { UserResponse } from '@/server/models/okta/User';
+import { UserResponse, UserUpdateRequest } from '@/server/models/okta/User';
 import { logger } from '@/server/lib/serverSideLogger';
 import { trackMetric } from '@/server/lib/trackMetric';
 
 /**
- * @method validateEmailAndPasswordSetSecurely
- *
- * Used to update the user has verified/validated their email and set a password securely.
+ * @name validateEmailAndPasswordSetSecurely
+ * @description Use this method to update the emailValidated and passwordSetSecurely flags for a user.
  *
  * @param {string} id accepts the Okta user ID, email address (login) or login shortname (as long as it is unambiguous)
  * @param {string} ip The IP address of the user
  * @param {boolean} flagStatus Default is true. If true, the flags are set to true, if false, the flags are set to false
+ * @param {boolean} updateEmail Default is true. If true, the emailValidated flag is updated to the flagStatus
+ * @param {boolean} updatePassword Default is true. If true, the passwordSetSecurely flag is updated to the flagStatus
  * @returns {Promise<UserResponse>} Promise that resolves to the user object
  */
 export const validateEmailAndPasswordSetSecurely = async ({
```

Comment:
non blocking, does this function name still make sense? Feels like its doing a lot more than just validating a user!